WordPress/Classes/ViewRelated/Feature Highlight/TooltipPresenter.swift

```diff
@@ -84,6 +84,8 @@ final class TooltipPresenter {
         }
     }
 
+    private var cons: [NSLayoutConstraint] = []
```

Comment:
Minor nitpick, but we usually prefer using descriptive variable names instead of shortened/abbreviated ones, and this extends to private properties. So perhaps this could be `tooltipConstraints` instead? And in the `setupConstraints()` method, you could use `self.` to refer to the instance variable.

Reply:
THis is totally forgotton and needs to be deleted. I was just testing things :(

Reply:
All good! 😄